examples/distributed/model.ts

```diff
@@ -4,9 +4,10 @@ const model_a = sm.network.remote_model('0.0.0.0:3001')
 const model_b = sm.network.remote_model('0.0.0.0:3002')
 
 const model = async (t: sm.Tensor) => {
-  t = await model_a(t)
-  t = await model_b(t)
-  return t
+  let res: sm.network.TFetchResult
+  res = await model_a(t)
+  res = await model_b(res.tensor)
+  return res.tensor
```

Comment:
today remote models have the nice property of

```javascript
import model from 'my_script'

const val = await model(t)
```

working exactly the same way as

```javascript
const model = sm.network.remote_model('0.0.0.0:3002')

const val = await model(t)
```

This is very much intentional for debugging ease.  I think we should preserve it.  To add stats, though, we can do this:

```javascript
const val = await model(t)
const stats = await model.stats
```

which will return `null` in the local model case.

Reply:
Now that we support the new `Stats` interface on a tensor I was able to restore the old interface `t = await model(t)`